EIPS/eip-4911.md

```diff
@@ -0,0 +1,157 @@
+---
+eip: 4911
+title: Composability Extension For ERC-721 Standard
+description: An extension of the ERC-721 standard to enable NFT tokens to own other NFT tokens
+author: Muhammed Emin Aydın (@muhammedea), Dmitry Savonin (@dmitry123)
+discussions-to: https://ethereum-magicians.org/t/erc-4911-composability-extension-for-erc-721-standard/8304
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-02-15
+requires: 165, 721
+---
+
+# Composability Extension For ERC-721 Standard
+
+## Abstract
+
+This standard outlines a smart contract interface that should be applied to ERC721 contracts to add composability feature.
```

Comment:
```suggestion
This standard outlines a smart contract interface that should be applied to [EIP-721](./EIP-721.md) contracts to add composability feature.
```

See [EIP-1 § Linking to other EIPs](https://eips.ethereum.org/EIPS/eip-1#linking-to-other-eips)